test/unittests/api/test_api.py

```diff
@@ -351,7 +351,7 @@ def test_is_paired_true(self, mock_request, mock_identity_get):
     @mock.patch('mycroft.api._paired_cache', False)
     @mock.patch('mycroft.api.IdentityManager.get')
     @mock.patch('mycroft.api.requests.request')
-    def test_is_paired_false(self, mock_request, mock_identity_get):
+    def test_is_paired_false_local(self, mock_request, mock_identity_get):
```

Comment:
does this test pass?  I thought you had to put an argument in the method definition for each patch.  same for the other test.  also, if you find yourself repeating a bunch of mocks like this, you can define them in setup and kill them in teardown.

Reply:
It does pass, I can't quite remember but I think it's because the _paired_cache one is set to False directly in the decorator.

I'll do some refactoring moving common mocks into a setup/teardown...Might need to restructure a bit to get it sane though